docs/concepts/abstractions/pod.md

```diff
@@ -0,0 +1,81 @@
+* **Pods that run multiple containers that need to work together**. A Pod might encapsulate an application that relies on multiple co-located containers that are tightly coupled and need to share resources. These co-located containers might form a single cohesive unit of service--one container serving files from a shared volume to the public, while a separate "sidecar" container refreshes or updates those files. The Pod wraps these containers and storage resources together as a single managable entity.
+
+The [Kubernetes Blog](http://blog.kubernetes.io) has some additional information on Pod use cases. For more information, see:
+
+* [The Distributed System Toolkit: Patterns for Composite Containers](http://blog.kubernetes.io/2015/06/the-distributed-system-toolkit-patterns.html)
+* [Container Design Patterns](http://blog.kubernetes.io/2016/06/container-design-patterns.html)
+
+Each Pod is meant to run a single instance of a given application. If you want to scale your application horizontally (e.g., run muliple instances), you should use multiple Pods, one for each instance. In Kubernetes, such Pods are usually managed by a Controller. See [Pods and Controllers](#pods-and-controllers) for more information.
+
+> Note: Restarting a container in a Pod should not be confused with _restarting_ the Pod itself.  The Pod itself does not run, but is an environment the containers run in and persists until it is deleted.
+
+#### How Pods Manage Multiple Containers
+
+Pods are designed to support multiple cooperating processes (as containers) that form a cohesive unit of service. The containers in a Pod are automatically co-located and co-scheduled on the same phyiscal or virtual machine in the cluster. The containers can share resources and dependencies, communicate with one another, and coordinate when and how they are terminated.
+
+Note that grouping multiple co-located and co-managed containers in a single Pod is a relatively advanced use case. You should use this pattern only in specific instances in which your containers are tightly coupled. For example, you might have a container that acts as a web server for files in a shared volume, and a separate "sidecar" container that updates those files from a remote source, as in the following diagram:
+
+![pod diagram](/images/docs/pod.svg){: style="max-width: 50%" }
+
+Pods provide two kinds of shared resources for their constituent containers: *networking* and *storage*.
+
+##### Networking
+
+Each Pod is assigned a unique IP address. Every the container in a pod shares the network namespace, including the IP address and network ports. Containers *inside a Pod* can communicate with one another using `localhost`. When containers in a Pod communicate with entities *outside the Pod*, they must coordinate how they use the shared network resources (such as ports).
```

Comment:
Is it worth saying:

Note: Containers encapsulated in the same Pod cannot bind to the same port, whereas containers running in different Pods can.